test/parallel/test-repl-unsupported-option.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const common = require('../common');
+const child_process = require('child_process');
+const assert = require('assert');
+
+const proc = child_process.spawn(process.execPath, ['-i', '--input-type=module']);
```

Comment:
Could you use `spawnSync()` to simplify the test?

Reply:
@cjihrig That makes sense. Thank you!